script/main.py

```diff
@@ -252,39 +252,35 @@ def most_popular_cities(conn, number):
   cur = conn.cursor()
   command = "SELECT location_city, count(location_city) FROM users GROUP BY location_city ORDER BY count(location_city) DESC LIMIT ?"
   cur.execute(command, (number,))
-  cities = cur.fetchall()
-  for city in cities:
-    city_name = city[0]
-    city_occurrences = city[1] 
-    print(f'City {city_name} occurr {city_occurrences} times.')
+  cities = cur.fetchall()  
+  return cities
   
 
 def most_common_passwords(conn, number):
   cur = conn.cursor()
   command = "SELECT login_password, COUNT(login_password) FROM users GROUP BY login_password ORDER BY COUNT(login_password) DESC limit ?"
   cur.execute(command, (number,))
   passwords = cur.fetchall()
```

Comment:
Ok. You can do it this way, but you can't return the iterator returned by `fetchall()`
You need to convert it to list first